data/gala.gschema.xml

```diff
@@ -15,7 +15,20 @@
 		<value nick='grid' value='0'/>
 		<value nick='natural' value='1'/>
 	</enum>
-	
+
+	<schema path="/io/elementary/desktop/screensaver/" id="io.elementary.desktop.screensaver/">
```

Comment:
Is that slash supposed to be on the end of the ID?

Reply:
I believe so. It looks like that's how the other schema paths are done

Reply:
I was looking at the `id` parameter, not the `path` one.